src/hotspot/share/logging/logDecorators.cpp

```diff
@@ -45,6 +49,13 @@ const char* LogDecorators::_name[][2] = {
 #undef DECORATOR
 };
 
+const LogDecorators::DefaultUndecoratedSelection LogDecorators::default_decorators[] = {
+#define UNDECORATED_DEFAULT(level, ...) LogDecorators::DefaultUndecoratedSelection(level, __VA_ARGS__),
+  UNDECORATED_DEFAULTS
+#undef UNDECORATED_TAGSET
```

Comment:
```suggestion
#undef UNDECORATED_DEFAULT
```
Typo, I think this was ment to match with the `#define`.